flowetl/etl/etl/production_postload_queries.py

```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Contains the definition of callables to be used in the production ETL dag.
+"""
+from sqlalchemy.orm.session import Session
+from pendulum.date import Date as pendulumDate
+
+
+# pylint: disable=unused-argument
+def num_total_calls__callable(*, cdr_date: pendulumDate, session: Session, **kwargs):
+    """
+    Function to determine the number of total calls
+    """
+
+    sql = f"""
+    select
```

Comment:
Tiny stylistic note, just for consistency with other code (mainly in flowmachine) where we typically write SQL keywords in uppercase.
```suggestion
    SELECT
```

Reply:
Hah! Would usually use uppercase for the SQL keywords, but when debugging the integration tests in `test_full_pipeline` for another branch saw that all the SQL code there was lowercase, so mistook that when being consistent 😄

Reply:
Haha, it may have been my own bias towards uppercase shining through here as well. 😉